frontend/src/components/CostTableMenu.vue

```diff
@@ -161,7 +158,7 @@ const handleUpdateRegion = (val: any[]) => {
   emit("update-region", val);
 };
 
-const handleUpdateChargeType = (val: ChargeType) => {
+const handleUpdateChargeType = (val: any) => {
```

Comment:
Is it intentionally to use `any` here?

Reply:
I just don't know how to deal with the type assertion. 
`Can not allocate type “(val: string[]) => void” to type “MaybeArray<(value: (string | number)[]) => void> | undefined”。`

Reply:
That's okay. Let's find a way later.